duckduckgo/iOS
# Don't crash when AppRatingPromptEntity fetch errors

Task/Issue URL: https://app.asana.com/0/0/1206441184793818/f

**Description**:

Removes a fatal error when we try to fetch the `AppRatingPromptEntity` and it errors. We might want to add a Pixel here, but there’s a silent recovery from the case that the fetch yields no results, so maybe we’re fine to just do the same here?

**Steps to test this PR**:
This is hard to reproduce and the fix doesn’t seem like it could go wrong. However perhaps someone could sanity check this for me in case there’s a reason we’d want to crash or at least make a bit more noise when this happens

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
